support/src/main/java/se/inera/intyg/common/support/facade/util/TestabilityToolkit.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.support.facade.util;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValue;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueDateRange;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueDateRangeList;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueDiagnosis;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueDiagnosisList;
+import se.inera.intyg.common.support.modules.support.facade.FillType;
+
+public class TestabilityToolkit {
+
+    private static final int DEFAULT_SICK_LEAVE_LENGTH = 14;
+    private static final int DEFAULT_SHORT_SICK_LEAVE_LENGTH = 4;
+
+    public static void fillCertificateWithTestData(Certificate certificate, FillType fillType,
+        TestabilityTestdataToolkit testabilityTestdataToolkit) {
+        if (FillType.MINIMAL.equals(fillType)) {
+            final var minimumValuesLisjp = testabilityTestdataToolkit.getMinimumValues();
```

Comment:
Should we change this to minimumValues & maximumValues?

Reply:
Nice catch!